scripts/upload-to-snack.ts

```diff
@@ -0,0 +1,196 @@
+import { Snack } from "snack-sdk";
+import { promises as fs } from "fs";
+import path from "path";
+import uiPackageJson from "../packages/ui/package.json";
+
+const PATHS = require("./paths");
+
+const UI_VERSION = uiPackageJson.version;
+const APP_JS_PATH = path.join(PATHS.EXAMPLE_ROOT, "App.js");
+const APP_SRC_PATH = path.join(PATHS.EXAMPLE_ROOT, "src");
+// const APP_FONTS = path.join(PATHS.EXAMPLE_ROOT, "src", "assets", "fonts");
```

Comment:
did you want to keep these?

Reply:
I see they're used by other commented code referenced below -- maybe copy the comment? idk